requirements.txt

```diff
@@ -6,4 +6,5 @@ pytest-cov==2.6.1
 capturer==2.4
 attrs==19.1.0
 requests==2.24.0
-pymongo==3.11.0
\ No newline at end of file
+pymongo==3.11.0
+redispy==3.0.0
```

Comment:
Don't you need to add it to setup.py as well?

Reply:
nope - that's just a dev dependency!